packages/instantsearch.js/src/connectors/voice-search/__tests__/connectVoiceSearch-test.ts

```diff
@@ -472,6 +472,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/voice-searc
         new SearchParameters({
           ignorePlurals: true,
           removeStopWords: true,
+          // @ts-ignore we send optionalWords as a string
           optionalWords: 'query',
```

Comment:
are both possible? if so, we can fix the spec

Reply:
yes, it's allowed in the api, but I think that's only because any array is also allowed to be a string

Reply:
ok ok so let's keep it like that then